src/modules/newTask.js

```diff
@@ -0,0 +1,99 @@
+import { setData } from './localStorage.js';
+import { toDo } from '../index.js';
```

Comment:
I feel the index.js should not be exporting as it is the main file, rather it should be importing all the functions.